contracts/core/EntryPointSimulations.sol

```diff
@@ -159,4 +159,10 @@ contract EntryPointSimulations is EntryPoint, IEntryPointSimulations {
         // always revert
         revert("");
     }
+
+    //make sure depositTo cost is more than normal EntryPoint's cost.
+    // empiric test showed that without this wrapper, simulation depositTo costs less..
```

Comment:
Any idea why it costs less? It literally calls the same function that gets called on-chain in EntryPoint.  If the way EntryPointSimulations calls it is cheaper, we should do the same in EntryPoint :)

Reply:
probably some optimization of the "VMT".. because we added more functions, it did some different "grouping" which caused this method to be little more efficient...